packages/broker/test/integration/plugins/operator/MaintainOperatorValueService.test.ts

```diff
@@ -0,0 +1,159 @@
+            fetch,
+            signer: operatorWallet
+        }
+        return { operatorWallet, operatorContract }
+    }
+
+    beforeEach(async () => {
+        provider = new JsonRpcProvider(chainURL)
+        logger.debug("Connected to: ", await provider.getNetwork())
+
+        adminWallet = new Wallet(adminPrivKey, provider)
+
+        token = new Contract(config.contracts.LINK, tokenABI, adminWallet) as unknown as IERC677
+        const timeString = (new Date()).getTime().toString()
+        const streamPath1 = "/operatorvalueservicetest-1-" + timeString
+        const streamPath2 = "/operatorvalueservicetest-2-" + timeString
+        streamId1 = adminWallet.address.toLowerCase() + streamPath1
+        streamId2 = adminWallet.address.toLowerCase() + streamPath2
+        const streamRegistry = new Contract(config.contracts.StreamRegistry, streamRegistryABI, adminWallet) as unknown as StreamRegistry
+        logger.debug(`creating stream with streamId1 ${streamId1}`)
+        await (await streamRegistry.createStream(streamPath1, "metadata")).wait()
+        logger.debug(`creating stream with streamId2 ${streamId2}`)
+        await (await streamRegistry.createStream(streamPath2, "metadata")).wait();
+        
+        ({ operatorWallet, operatorContract } = await deployNewOperator())
+    })
+
+    it("updates both sponsorships to stay over the threshold", async () => {
+        await (await token.connect(operatorWallet).transferAndCall(operatorContract.address, parseEther("200"), operatorWallet.address)).wait()
+        // deploy 2 sponsorships, sponsor 200 & stake 100 into both of them
+        const sponsorship1 = await deploySponsorship(config, operatorWallet, { streamId: streamId1 })
+        const sponsorship2 = await deploySponsorship(config, operatorWallet, { streamId: streamId2 })
+        await (await token.connect(operatorWallet).transferAndCall(sponsorship1.address, parseEther("200"), "0x")).wait()
+        await (await token.connect(operatorWallet).transferAndCall(sponsorship2.address, parseEther("200"), "0x")).wait()
+        await (await operatorContract.stake(sponsorship1.address, parseEther("100"))).wait()
+        await (await operatorContract.stake(sponsorship2.address, parseEther("100"))).wait()
+
+        expect(await token.balanceOf(sponsorship1.address)).toEqual(parseEther("300")) // 200 sponsored + 100 staked
+        expect(await token.balanceOf(sponsorship2.address)).toEqual(parseEther("300"))
+        
+        const maintainOperatorValueService = new MaintainOperatorValueService(opertatorConfig)
+
+        const totalValueInSponsorshipsBefore = await operatorContract.totalValueInSponsorshipsWei()
+        const penaltyFraction = 0.0005 // * 1e18
+        const threshold = 200 * penaltyFraction // 0.1
+
+        // wait for sponsorships to accumulate earnings so approximate values differ enough form the real values
+        await new Promise((resolve) => setTimeout(resolve, 3000)) // sleep 3 sec
```

Comment:
We have a utility for this in `@streamr/utils` package
```suggestion
        await wait(3000)
```